src/Reformat.js

```diff
@@ -20,18 +20,22 @@
  *          work with
  **************************************************************/
 function to_JSON(input){
-    var rxo = {
+    var rxo = {//^(\-|\+){0,1}\w{1,}(?!:)
         /* jshint ignore:start */
         "is_blank": /^\W{0,}$/,
         "is_objective": /(max|min)(imize){0,}\:/i,
-        "is_int": /^\W{0,}int/i,
+        //previous version
+        //"is_int": /^\W{0,}int/i,
+        //new version to avoid comments
+        "is_int": /^(?!\/\*)\W{0,}int/i,
+        "constraint_element": /(\-|\+){0,1}\w{0,}/i,
         "is_constraint": /(\>|\<){0,}\=/i,
         "is_unrestricted": /^\S{0,}unrestricted/i,
         "parse_lhs":  /(\-|\+){0,1}\s{0,1}\d{0,}\.{0,}\d{0,}\s{0,}[A-Za-z]\S{0,}/gi,
         "parse_rhs": /(\-|\+){0,1}\d{1,}\.{0,}\d{0,}\W{0,}\;{0,1}$/i,
         "parse_dir": /(\>|\<){0,}\=/gi,
         "parse_int": /[^\s|^\,]+/gi,
-        "get_num": /(\-|\+){0,1}(\W|^)\d+\.{0,}\d{0,}/g,
+        "get_num": /(\-|\+){0,1}(\W|^)\d+\.{0,1}\d{0,}/g, // Why accepting character \W before the first digit?
```

Comment:
@JWally , we were wondering what the ```\W``` was for? Does it means that if a character (other than an alphabet or a number) appears between the sign and the first digit, we still consider the word as a number? In which case does it happen?